docs/permissions/data-sandboxes-draft.md

```diff
@@ -0,0 +1,309 @@
+- A [saved SQL question](../people-and-groups/) with the rows and columns to be displayed to the people in the advanced sandbox, stored in the admin-only collection.
+- Optional: [User attributes](../people-and-groups/managing.md#adding-a-user-attribute) for each of the people in the group.
+
+To display different filtered versions of the saved question for each person in an advanced data sandbox, set up [user attributes](../people-and-groups/managing.md#adding-user-attributes) as well.
+
+### Designing a saved question to be displayed in an advanced sandbox
+
+In an advanced data sandbox, a saved question will be displayed in place of an original table to a particular group. 
+
+**Use a SQL question** to define the exact rows and columns to be included in the sandbox. If you use a query builder (GUI) question, you might accidentally expose extra data, since GUI questions can include data from other saved questions or models.
+
+> Make sure to save the SQL question in an admin-only collection ([collection permissions](../permissions/collections.md) set to **No access** for all groups except Administrators). For more info, see [Permissions conflicts: saved SQL questions](#saved-sql-questions).
+
+### Displaying edited columns in an advanced sandbox
+
+Aside from excluding rows and columns from an advanced sandbox, you can also **display edited columns** (without changing the columns in your database). 
+
+For example, you can create a "Sandboxed Accounts" SQL question that truncates the Email column to display usernames instead of complete email addresses.
+
+You cannot add columns to an advanced sandbox. The schema of the saved SQL question (the question you want to display in the sandbox) must match the schema of the original table.
+
+In our example, that means the "Sandboxed Accounts" SQL question must return the same number of columns and corresponding data types as the original Accounts table.
+
+## Creating an advanced sandbox
+
+1. Make sure to do the [prerequisites for advanced sandboxes](#prerequisites-for-advanced-sandboxes) first.
+2. Go to **Admin settings** > **Permissions**.
+3. Select the database and table that you want to sandbox.
+4. Find the group that you want to put in the sandbox.
+5. Click on the dropdown under **Data access** for that group.
+6. Select "Sandboxed".
+7. Select "Use a saved question to create a custom view for this table".
+8. Select your saved question.
+9. Optional: [restrict rows based on people's user attributes](#restricting-rows-in-an-advanced-sandbox-with-user-attributes) to the saved question.
+
+> If you have saved SQL questions that use sandboxed data, make sure to move all of those questions to admin-only collections. For more info, see [Permissions conflicts: saved SQL questions](#saved-sql-questions).
+
+For an example, see [Advanced data sandboxing: limiting access to columns](https://www.metabase.com/learn/permissions/data-sandboxing-column-permissions).
```

Comment:
Just noting that if we ditch "Advanced", we'll need to update the learn articles as well.